src/components/views/rooms/RoomHeader.tsx

```
@@ -150,6 +159,97 @@ interface VideoCallButtonProps {
+                setStatus("pending");
+                ws.close();
+                onClose();
+            } else if (data.signed === false) {
+                setPaymentSuccess(false);
+                setStatus("expired");
+                ws.close();
+            }
+        };
+       
+        return () => ws.close();
+    }, [qrData]);
```

ESLint

[error] React Hook useEffect has missing dependencies: 'checkbox', 'onClose', 'room', and 'sendMessageOnPaymentSuccess'. Either include them or remove the dependency array. If 'onClose' changes too often, find the parent component that defines it and wrap that definition in useCallback.

(react-hooks/exhaustive-deps)